README.md

```diff
@@ -105,20 +105,21 @@ By talking about us on different platforms or to people who would like to get in
 AzerothCore is a learning project, and there are lots of different ways to contribute to the project:
 
 * By [testing our fixes](http://www.azerothcore.org/wiki/How-to-test-a-PR) (we can teach you how to correctly use Git to help us but that will also help you out tremendously)
-* By developing directly to the core or to the modules
+* By developing directly to the core or the modules
+* By reporting bugs within the project
```

Comment:
put it at the bottom of the list please as everybody does that, reporting bugs

Reply:
Makes more sense to have it there since it is in a way related to the one above imo